crates/frontend/src/pages/home.rs

```diff
@@ -50,6 +51,115 @@ fn gen_name_id(s0: &String, s1: &String, s2: &String) -> String {
     format!("{s0}::{s1}::{s2}")
 }
 
+#[component]
+fn all_context_view(config: Config) -> impl IntoView {
+    let Config {
+        contexts,
+        overrides,
+        default_configs,
+    } = config;
+    let rows = |k: &String, v: &Value, striked: bool| {
+        let default_iter = vec![(k.clone(), v.clone())];
+        v
+            .as_object()
+            .unwrap_or(&Map::from_iter(default_iter))
+            .iter()
+            .map(|(key, value)| {
+                let key = key.replace('"', "").trim().to_string();
+                let value = value
+                    .as_str()
+                    .unwrap_or(value.to_string().trim_matches('"'))
+                    .into();
+                let unique_name = gen_name_id(&key, &key, &value);
+                view! {
+                    <tr>
+                        <td class="min-w-48 max-w-72 font-mono">
+                            <span
+                                name=format!("{unique_name}-1") class="config-name"
+                                class:text-black={!striked}
+                                class:font-bold={!striked}
+                                class:text-gray-300={striked}
+                            >{key}</span>
+                        </td>
+                        <td class="min-w-48 max-w-72 font-mono" style="word-break: break-word;">
```

Comment:
I mostly just moved the code out of that big component function this just an old code when we were building the first cut for UI and never it touched it again, but sure I will make the change since we already made a class for that style.